docs/configuring-playbook-own-webserver.md

```diff
@@ -26,4 +35,95 @@ nextcloud_nginx_proxy_enabled: false
 
 - ensure that the `/.well-known/acme-challenge` location for the "port=80 vhost" gets proxied to `http://localhost:2403` (controlled by `nextcloud_ssl_certbot_standalone_http_port`) for automated SSL renewal to work
 
-- ensure that you restart/reload your webserver once in a while, so that renewed SSL certificates would take effect (once a month should be enough)
\ No newline at end of file
+- ensure that you restart/reload your webserver once in a while, so that renewed SSL certificates would take effect (once a month should be enough)
+
+## Method 2: Fronting the integrated nginx reverse-proxy webserver with another reverse-proxy
+
+This method is about leaving the integrated nginx reverse-proxy webserver be, but making it not get in the way (using up important ports, trying to retrieve SSL certificates, etc.).
+
+If you wish to use another webserver, the integrated nginx reverse-proxy webserver usually gets in the way because it attempts to fetch SSL certificates and binds to ports 80 and 443.
+
+You can disable such behavior and make the integrated nginx reverse-proxy webserver only serve traffic locally (or over a local network).
+
+You would need some configuration like this:
+
+### Sample configuration for running behind Traefik 2.0
+
+Below is a sample configuration for using this playbook with a [Traefik](https://traefik.io/) 2.0 reverse proxy.
+
+```yaml
+# Disable generation and retrieval of SSL certs (Traefik will handle this)
+nextcloud_ssl_retrieval_method: none
+
+# All containers need to be on the same Docker network as Traefik
+# (This network should already exist and Traefik should be using this network)
+nextcloud_docker_network: 'traefik'
+
+nextcloud_nextcloud_docker_container_extra_arguments:
+  # May be unnecessary depending on Traefik config, but can't hurt
+  - '--label "traefik.enable=true"'
+
+  # The Nginx proxy container will receive traffic from these subdomains
+  - '--label "traefik.http.routers.nextcloud-nginx-proxy.rule=Host(`{{ host_specific_nextcloud_hostname }}`)"'
+
+  # (The 'web-secure' entrypoint must bind to port 443 in Traefik config)
+  - '--label "traefik.http.routers.nextcloud-nginx-proxy.entrypoints=web-secure"'
+
+  # (The 'default' certificate resolver must be defined in Traefik config)
+  - '--label "traefik.http.routers.nextcloud-nginx-proxy.tls.certResolver=default"'
```

Comment:
Even with this, `nextcloud-nginx-proxy` would sitll be installed and would still try to occupy ports `80` and `443`, won't it? We'd need something else added here (and to the playbook) to disable it.

Reply:
That's a good point! I'll take a look and revisit this PR with changes that reflect this suggestion.